cvat-ui/src/components/annotation-page/attribute-annotation-workspace/attribute-annotation-sidebar/attribute-annotation-sidebar.tsx

```diff
@@ -2,32 +2,28 @@
 //
 // SPDX-License-Identifier: MIT
 
-import React, { useState, useEffect } from 'react';
-import GlobalHotKeys, { KeyMap } from 'utils/mousetrap-react';
-import { connect } from 'react-redux';
+import { MenuFoldOutlined, MenuUnfoldOutlined } from '@ant-design/icons';
 import Layout, { SiderProps } from 'antd/lib/layout';
 import { SelectValue } from 'antd/lib/select';
-import { Row, Col } from 'antd/lib/grid';
 import Text from 'antd/lib/typography/Text';
-import { MenuFoldOutlined, MenuUnfoldOutlined } from '@ant-design/icons';
-
-import { ThunkDispatch } from 'utils/redux';
-import { Canvas } from 'cvat-canvas-wrapper';
-import { LogType } from 'cvat-logger';
+import React, { useEffect, useState } from 'react';
```

Comment:
PR looks good. Only ask to normilize imports here and in `object-list-header.tsx`